### DISCUSSION TITLE
Support Filament Panel's Subheading Resource

### DISCUSSION THREAD

**Discussion by elliottschoolcraft** [awcodes/filament-curator#474] [Ideas]
At the moment, the current set of resources allowed by curator limits us from being able to apply a subheading in the media panel, however, Filament's panels allow for a subheading to be added.

https://filamentphp.com/docs/3.x/panels/pages#adding-a-page-subheading
Filament example: protected ?string $subheading = 'Manage and activate different modules or features.';

Curator Update:
    'resources' => [
        'label' => 'Media',
        'plural_label' => 'Media',
        'subheading' => 'Manage and activate different modules or features.',
        'navigation_group' => 'Website',
        'navigation_icon' => 'heroicon-o-photo',
        'navigation_sort' => null,
        'navigation_count_badge' => false,
        'resource' => \Awcodes\Curator\Resources\MediaResource::class,
    ],

**Comment by awcodes** [comment]
Just extend Curator's resource and set the config to use your resource. Then you'll have full control over it.
